requirements.txt

```diff
@@ -1 +1,7 @@
-cryptography
+cffi==1.8.3
```

Comment:
This is `pip freeze` output. Shouldn't we at least pin major versions? Apparently, we can do this: https://www.python.org/dev/peps/pep-0440/#compatible-release